.travis.yml

```diff
@@ -2,6 +2,7 @@ dist: trusty
 
 before_install:
   - sudo apt-get install coreutils realpath doxygen graphviz python-lesscpy cppcheck coccinelle pcregrep python3-flake8
+  - which flake8
```

Comment:
I assume that you are experimenting on Travis with these commits.

Reply:
yes, it doesn't work because python 2 flake8. I'm trying to understand what's going on there.

Reply:
see https://packages.ubuntu.com/trusty/python3-flake8 and https://packages.ubuntu.com/trusty/amd64/python3-flake8/filelist for some hints

Reply:
Thanks, it helped. Actually python3-flake8 has a dependency to python-flake8. The python2 flake8 was taken into account and that's why there was a issue with `ProcessLookupError`: it only exists with python3.

Installing flake8 using pip3 fixes the problem. That's certainly why I did that this way at the very beginning of the work on adding flake8 check to static tests.